docs/ops/detection/Proposal_4.md

```diff
@@ -0,0 +1,187 @@
+## Proposal <a name="Proposal"></a>
+
+**Versioned name**: *Proposal-4*
+
+**Category**: *Object detection*
+
+**Short description**: *Proposal* operation filters bounding boxes and outputs only those with the highest prediction confidence.
+
+**Detailed description**
+
+*Proposal* has three inputs: a tensor with probabilities whether particular bounding box corresponds to background and foreground, a tensor with logits for each of the bounding boxes, a tensor with input image size in the [`image_height`, `image_width`, `scale_height_and_width`] or [`image_height`, `image_width`, `scale_height`, `scale_width`] format. Operation produces two tensors: the first mandatory tensor of shape `[batch_size * post_nms_topn, 5]` with proposed boxes and the second optional tensor of shape `[post_nms_topn]` with probabilities.
+*Proposal* layer does the following with the input tensor:
+1.  Generates initial anchor boxes. Left top corner of all boxes is at (0, 0). Width and height of boxes are calculated from *base_size* with *scale* and *ratio* attributes.
+2.  For each point in the first input tensor:
+    *   pins anchor boxes to the image according to the second input tensor that contains four deltas for each box: for *x* and *y* of center, for *width* and for *height*
+    *   finds out score in the first input tensor
+3.  Filters out boxes with size less than *min_size*
+4.  Sorts all proposals (*box*, *score*) by score from highest to lowest
+5.  Takes top *pre_nms_topn* proposals
+6.  Calculates intersections for boxes and filter out all boxes with \f$intersection/union > nms\_thresh\f$
+7.  Takes top *post_nms_topn* proposals
+8.  Returns top proposals and optionally their probabilities 
```

Comment:
please check that this part is actually rendered as a list

Reply:
Checked, it renders as expected.